README.md

```diff
@@ -1,2 +1,110 @@
-# Air-Hockey-Robot
-Srishti 2020
+# AIR HOCKEY ROBOT
+### Shrishti'20, IITR
+## Image of the project
+1. Table isometric view 
+![Table isometric view 1](https://user-images.githubusercontent.com/68807083/88667192-a8c56200-d0fe-11ea-8249-519fb2419034.jpg)
+
+## Abstract
+<p align="justify">To create an autonomous gaming system of air hockey where one player is human and other side will be played by a bot. The bot will automatically detect the position of the moving puck at every instant and defend or strike accordingly. The aim is to set up difficulty levels as well whose control will be subject to the human player.
```

Comment:
When you start a paragraph with <p> tag you should end it with <\p>. For example `<p align="justify">Sample Text<\p>`. Do this change throughout README.md